src/test/java/org/influxdb/InfluxDBTest.java

```diff
@@ -323,10 +326,10 @@ public void testWriteMultipleStringDataLinesThroughUDP() {
         Query query = new Query("SELECT * FROM " + measurement + " GROUP BY *", UDP_DATABASE);
         QueryResult result = this.influxDB.query(query);
 
-        Assert.assertEquals(3, result.getResults().get(0).getSeries().size());
-        Assert.assertEquals(result.getResults().get(0).getSeries().get(0).getTags().get("atag"), "test1");
-        Assert.assertEquals(result.getResults().get(0).getSeries().get(1).getTags().get("atag"), "test2");
-        Assert.assertEquals(result.getResults().get(0).getSeries().get(2).getTags().get("atag"), "test3");
+        Assertions.assertEquals(3, result.getResults().get(0).getSeries().size());
+        Assertions.assertEquals(result.getResults().get(0).getSeries().get(0).getTags().get("atag"), "test1");
```

Comment:
(minor adjustment) the "expected value" should be the first parameter

Reply:
good  catch will fix.